src/Client/ContentHubServiceVersion1.php

```diff
@@ -140,9 +140,9 @@ public function getWebhooksStatus(): array {
    * {@inheritDoc}
    */
   public function purge(): array {
-    $response = \Drupal::service('acquia_contenthub.client_manager')->createRequest('purge');
+    $response = $this->client->purge();
     if (!isset($response['success']) || $response['success'] !== TRUE) {
-      $error_message = isset($response['error']['message']) ? $response['error']['message'] : $response['error'];
+      $error_message = $response['error']['message'] ?? $response['error'] ?? $response;
```

Comment:
Not convinced this logic suffices here. In the following line we're treating the variable as a string, which is a false assumption. Let's convert it into string using `printf`.